src/main/java/com/crowdin/client/applications/ApplicationsApi.java

```diff
@@ -96,4 +96,71 @@ public ResponseObject<Map<String, Object>> editApplicationData(String applicatio
         ApplicationDataResponseObject response = this.httpClient.patch(builtUrl, request, new HttpRequestConfig(), ApplicationDataResponseObject.class);
         return ResponseObject.of(response.getData());
     }
+
+    /**
+     * @param identifier identifier of the application installation
+     * @return application installation data
+     * @see <ul>
+     * <li><a href="https://developer.crowdin.com/api/v2/#operation/api.applications.installations.get" target="_blank"><b>API Documentation</b></a></li>
+     * <li><a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.applications.installations.get" target="_blank"><b>Enterprise API Documentation</b></a></li>
+     * </ul>
+     */
+    public ResponseObject<Map<String, Object>> getApplicationInstallation(String identifier) throws HttpException, HttpBadRequestException {
+        String builtUrl = String.format("%s/applications/installations/%s", this.url, identifier);
+        ApplicationDataResponseObject response = this.httpClient.get(builtUrl, new HttpRequestConfig(), ApplicationDataResponseObject.class);
```

Comment:
this is the wrong response object here and below. You need to create a separate one `ApplicationInstallationResponseObject`